src/target_info.rs

```diff
@@ -65,15 +65,21 @@ fn check_processor_target_compatability(cores: &[Core], elf_path: &Path) {
     let matches = match core_type {
         CoreType::Armv6m => target == "thumbv6m-none-eabi",
         CoreType::Armv7m => target == "thumbv7m-none-eabi",
+        CoreType::Armv7a => target == "thumbv7a-none-eabi",
```

Comment:
Does `thumbv7a-none-eabi` even exist (yet)? I can't find it in the platform list: https://doc.rust-lang.org/nightly/rustc/platform-support.html

Reply:
Oops, you are right, that would probably be `armv7a-none-eabi` or `armv7a-none-eabihf`.
But given that I didn't test it on armv7a at all (and don't have suitable hardware to do so) I wonder if it would be better to handle it like Armv8a and print a warning, "Unsupported architecture"?

Reply:
I think that is reasonable. We have some cortex-m specific code anyways.